example/react/Readme.md

```diff
@@ -0,0 +1,14 @@
+# Casbin.js in React
```

Comment:
This library should be kept as small as possible, so no example code here. The examples can be put into: https://github.com/casbin-js

Reply:
I will separate this example into an independent repo.